app/views/content_items/service_sign_in/_choose_sign_in.html.erb

```diff
@@ -27,6 +27,7 @@
   <%= render "govuk_publishing_components/components/fieldset", legend_text: legend_text do %>
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-two-thirds">
+      <div id="verify-hint" data-module="show-gov-uk-verify-hint" style="display:none"></div>
```

Comment:
`GOVUK.modules` should be triggered with a `data-module='moduleName'`, rather than triggering them by passing an element into the constructor. This has an added benefit of preventing the same module from running twice on a single element.

When [`GOVUK.modules.start`](https://github.com/alphagov/govuk_publishing_components/blob/b9b46987c8e1800fcf1f392dec404846cc7d171e/app/assets/javascripts/govuk_publishing_components/modules.js#L25) is called, it runs through the DOM looking for every element that has a `data-module` attribute. If the value of that attribute corresponds to a module that's available, it then initialises the module.

```suggestion
      <div id="verify-hint" data-module="show-gov-uk-verify-hint" style="display:none" data-module="show-gov-uk-verify-hint"></div>
```

Reply:
that's what it's doing already?

Reply:
It is. Apologies!